code/drasil-data/lib/Data/Drasil/Quantities/Physics.hs

```diff
@@ -44,82 +44,82 @@ acceleration, angularAccel, angularDisplacement, angularVelocity, chgInVelocity,
   yVel, momentum, moment, moment2D, fOfGravity, positionVec, tension, angularFrequency, 
   period, frequency, chgMomentum :: UnitalChunk
 
-acceleration         = ucs' CP.acceleration (vec lA) (Vect Real) accelU
-angularAccel         = uc CP.angAccel lAlpha angAccelU
-angularDisplacement  = uc CP.angDisp lTheta radian
-angularFrequency     = uc CP.angFreq cOmega second
-angularVelocity      = uc CP.angVelo lOmega angVelU
-chgInVelocity        = uc CP.chgInVelocity (Atop Delta $ vec lV) velU
-constAccel           = uc CP.constAccel (sup lA lC) accelU
-displacement         = uc CP.displacement (vec lU) metre
-distance             = uc CP.distance lD metre
-energy               = uc CP.energy cE joule
-force                = uc CP.force (vec cF) newton
-frequency            = uc CP.frequency lF hertz
-gravitationalAccel   = uc CP.gravitationalAccel (vec lG) accelU
-gravitationalConst   = uc CP.gravitationalConst cG gravConstU
-height               = uc CP.height lH metre
-impulseS             = uc CP.impulseS lJ impulseU
-impulseV             = uc CP.impulseV (vec cJ) impulseU
-kEnergy              = uc CP.kEnergy  (Concat [cK, cE]) joule
-linearAccel          = uc CP.linAccel (Concat [vec lA, label "(", lT, label ")"]) accelU
-linearDisplacement   = uc CP.linDisp  (Concat [vec lU, label "(", lT, label ")"]) metre
-linearVelocity       = uc CP.linVelo  (Concat [vec lV, label "(", lT, label ")"]) velU
-momentOfInertia      = uc CP.momentOfInertia (vec cI) momtInertU
-chgMomentum          = uc CP.chgMomentum (Concat [cDelta,vec cP]) impulseU
-momentum             = uc CP.momentum (vec cP) impulseU
-moment               = uc CP.moment   (vec cM) torqueU
-moment2D             = uc CP.moment   cM       torqueU
+acceleration         = uc CP.acceleration (vec lA) (Vect Real) accelU
```

Comment:
in general, when I have to edit these things, I then try hard to make things line up in columns. Makes the code nice for humans to read and see that it is really a "database"